packages/ldap-auth-backend/README.md

```diff
@@ -1 +1,187 @@
+-   [Configurations](#configurations)
+    -   [Connection Configuration](#connection-configuration)
+    -   [Setup Backstage official LDAP plugin](#setup-backstage-official-ldap-plugin)
+    -   [Add authentication backend](#add-authentication-backend)
+    -   [Add the login form](#add-the-login-form)
+-   [Powered Apps](#powered-apps)
+-   [Support & Contribute](#support--contribute)
+-   [License](#license)
+
+<!-- tocstop -->
+
+## Installation
+
+> These packages are available on npm.
+
+You can install them in your backstage installation using `yarn workspace`
+
+```bash
+# install yarn if you don't have it
+$ npm install -g yarn
+# install backend plugin
+$ yarn workspace backend add @immobiliarelabs/backstage-plugin-ldap-auth-backend
+# install frontend plugin
+$ yarn workspace app add @immobiliarelabs/backstage-plugin-ldap-auth
+```
+
+## Configurations
+
+> This documentation assumes that you have already scaffolded your Backstage instance from the official `@backstage/create-app`, all files that we're going to customize here are the one already created by the CLI!
+
+### Connection Configuration
+
+> Adds connection configuration inside your backstage YAML config file, eg: `app-config.yaml`
+
+```yml
+auth:
+    environment: ENV_NAME
+    providers:
+        ldap:
+            ENV_NAME:
+                url:
+                    - 'ldaps://123.123.123.123'
+                rejectUnauthorized: false
+                userDn: 'ou=usr,dc=ns,dc=frm'
+                userSearchBase: 'dc=ns,dc=frm'
+```
+
+### Setup Backstage official LDAP plugin
+
+If you didn't have already, we need to configure the official LDAP plugin to imports and keep in syncs users
+
+> `packages/backend/src/plugins/catalog.ts`
+
+```ts
+import type { Router } from 'express';
+import type { PluginEnvironment } from '../types';
+
+import { CatalogBuilder } from '@backstage/plugin-catalog-backend';
+import { ScaffolderEntitiesProcessor } from '@backstage/plugin-scaffolder-backend';
+import {
+  LdapOrgEntityProvider,
+} from '@backstage/plugin-catalog-backend-module-ldap';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  const builder = await CatalogBuilder.create(env);
+
+  builder.addEntityProvider(
+    LdapOrgEntityProvider.fromConfig(env.config, {
+      id: '<YOUR-ID>',
+      target: 'ldaps://<YOUR-ADDRESS>',
+      logger: env.logger,
+      schedule: env.scheduler.createScheduledTaskRunner({
+        frequency: // whatever
+        timeout: // whatever
+      }),
+    }),
+  );
+
+  builder.addProcessor(new ScaffolderEntitiesProcessor());
+  const { processingEngine, router } = await builder.build();
+  await processingEngine.start();
+  return router;
+}
+```
+
+### Add authentication backend
+
+This assumes a basic usage: single process without custom auth function or user object customization and in-memory token storage
+
+For more uses cases you can see the [example folders](../../examples/)
```

Comment:
maybe better `[example folders](https://github.com/immobiliare/backstage-plugin-ldap-auth/examples)` ?

Reply:
If we do absolute it always links to the main examples even if you're on another branch when you click this